requirements.txt

```diff
@@ -1,21 +1,26 @@
+# Package deps
 arrow
-requests>=2.0,<3.0dev
 pint
+pydantic
 pytz
+requests>=2.0,<3.0dev
```

Comment:
oh we should revisit this later to make sure we don't get into trouble with an upper limit dependency .